paimon-hive/paimon-hive-catalog/src/main/java/org/apache/paimon/hive/HiveCatalog.java

```diff
@@ -177,6 +177,12 @@ public void dropDatabase(String name, boolean ignoreIfNotExists, boolean cascade
         }
     }
 
+    @Override
+    public void renameDatabase(String fromDatabase, String toDatabase, boolean ignoreIfNotExists)
+            throws DatabaseNotExistException, DatabaseAlreadyExistException {
+        throw new UnsupportedOperationException("Hive catalog does not support rename database.");
```

Comment:
Why not support rename database of `HiveCatalog`? Just uses the `IMetaStoreClient#alterDatabase`?

Reply:
Hive does not support rename database.

Reply:
@s7monk, could you take a look at above method `IMetaStoreClient#alterDatabase`?